addon/components/ecsy-babylon/scene.ts

```diff
@@ -69,6 +76,15 @@ export default class EcsyBabylonScene extends DomlessGlimmerComponent<EcsyBabylo
     }
   }
 
+  rotateEnvironmentTexture(angle = 0) {
+    const texture = this.scene.environmentTexture;
+    texture.setReflectionTextureMatrix(
```

Comment:
tests say this might be null, apparently

Reply:
Hm, yes. Why did my IDE not complain!? 🤔

Reply:
Ah, sh*t. Did I mention that `any` really is the devil...

https://github.com/kaliber5/ember-ecsy-babylon/blob/master/addon/components/ecsy-babylon/scene.ts#L17

Reply:
should be fixed now!

Reply:
`any` really is